L/LLVMExtra/build_tarballs.jl

```diff
@@ -34,42 +39,69 @@ cmake -B build -S . -GNinja ${CMAKE_FLAGS[@]}
 ninja -C build -j ${nproc} install
 """
 
-function configure(julia_version, llvm_version)
-    # These are the platforms we will build for by default, unless further
-    # platforms are passed in on the command line
-    platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
-
-    foreach(platforms) do p
-        BinaryPlatforms.add_tag!(p.tags, "julia_version", string(julia_version))
-    end
+augment_platform_block = """
+    function augment_platform!(platform)
+        llvm_version = Base.libllvm_version
+
+        # does our LLVM build use assertions?
+        llvm_assertions = try
+            cglobal((:_ZN4llvm24DisableABIBreakingChecksE, Base.libllvm_path()), Cvoid)
+            false
+        catch
+            true
+        end
+
+        platform["llvm"] = if llvm_assertions
```

Comment:
This should be guarded by a `haskey` otherwise  you can't cross-instantiate with a `Pkg.instantiate(;platform=)`.